src/pages/Reservation/ReservUserInfo/ReservUserInfo.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PassengerInfo from './PassengerInfo/PassengerInfo';
+
+export default function ReservUserInfo() {
+  const [bookerName, setBookerName] = useState('');
+  const [bookerEmail, setBookerEmail] = useState('');
+  const [bookerNumber, setBookerNumber] = useState(0);
+
+  const validateName =
+    bookerName.length > 1 || bookerName === '' ? true : false;
+
+  const validateEmail =
+    bookerEmail.includes('@' && '.com') || bookerEmail === '' ? true : false;
+
+  const validateNumber =
+    bookerNumber.toString().length > 9 || bookerNumber === 0 ? true : false;
+
+  return (
+    <ReservUserInfoWrapper>
+      <p className="BookerInfoTitle">예약자 정보 (필수)</p>
+      <hr />
+      <BookerInput>
+        <label htmlFor="bookerName">예약자 이름</label>
+        <input
+          className={validateName ? 'validated' : 'unvalidated'}
+          type="text"
+          name="bookerName"
+          placeholder="홍길동"
+          onChange={e => setBookerName(e.target.value)}
+        />
+        <span className="noticeAboutInput" hidden={!validateName}>
+          ㅤ
+        </span>
+        <span className="validationHint" hidden={validateName}>
+          예약자 이름을 2자 이상 입력하세요.
+        </span>
+      </BookerInput>
+      <BookerInput>
+        <label htmlFor="bookerEmail">이메일 주소</label>
+        <input
+          className={validateEmail ? 'validated' : 'unvalidated'}
```

Comment:
```suggestion
        <Input
          isValid={validateEmail}
```
스타일드 컴포넌트를 쓰는 만큼 클래스명으로 토글해주지 않아도 다른 스타일을 만들어줄 수 있을거 같습니다